test/model/test_node.py

```diff
@@ -1515,20 +1515,20 @@ async def test_statistics_updated(
                 "rssi": 7,
                 "lwr": {
                     "protocolDataRate": 1,
-                    "repeaters": [wallmote_central_scene.node_id],
+                    "repeaters": [f"{wallmote_central_scene.node_id}"],
```

Comment:
Yeah I saw that - I'm doing the casting because I'm pretty sure the statistics keys changed from a string to an int at some point and I want to avoid having to patch this for every time one of these keys changes. Cc @AlCalzone the node key in the statistics updated event changed from an int to a string. Maybe in 12.x? Is this a bug or should we always assume these types of keys could be strings?

Reply:
Oh and it does not matter if you read a value using key `2` or `"2"`

Reply:
Got it so basically we do need to assume that keys could be in either form when they are numeric. Even if they aren't now as part of a refactor, optimization, or change on the driver side, they could switch to strings. We could probably be more precise but it's easier to assume they could all be strings

Reply:
yeah I take your point, however I want to make sure that anywhere where we expect an int, we get an int, so I am forcing the conversion to a string to make sure the casting is set up properly. I looked for all instances where we get a node ID in a JSON payload and made them strings so that I could be sure I found all of them